### DISCUSSION TITLE
Show a note of message being deleted

### DISCUSSION THREAD

**Discussion by alexInhert** [community/community#15776] [Pull Requests]
Some times in PR/issues people delete their messages but there is no record of such deletions
This causes for people who read the thread not to understand the flow.
My suggestion is: if someone delete the message place an event/other notice that a message was there and was removed.

**Comment by suprstarrd** [comment]
More specifically, there *is* a record of the deletion, just not in the spot of the message. This does break (and has broken) the flow of PRs and issues.
